app/Http/Controllers/HomeController.php

```diff
@@ -39,6 +46,10 @@ public function index()
                     'hotel' => $hotel,
```

Comment:
$hotel = $this->hotel->getPopularHotel(4);
$tour   = $this->tour->getPopularTour(8);
$car    = $this->car->getPopularCar(6);
=>> các tham 4 6 8 là cái gì. định nghĩa cho nó. k điền số như này. đây mà magic number